Makefile

```diff
@@ -4,8 +4,7 @@
 CC   =   g++
 CCVERSIONGTEQ48 := $(shell expr `g++ -dumpversion | cut -f1,2 -d.` \>= 4.8)
 
-#UCFLAGS = -O0 -g3 -Wall -gstabs+ -DSTANDALONE
-UCFLAGS = -O3 -Wall -DSTANDALONE
```

Comment:
`-DSTANDALONE` should be kept, it tells the preprocessor that it is the standalone compilation. Otherwise it probably won't compile